docs/fides/docs/development/releases.md

```diff
@@ -38,21 +39,129 @@ We use GitHub’s `release` feature to tag releases that then get automatically
 1. Auto-populate the release notes
 1. Publish the release
 
-### Minor
-1. Create a new branch from the latest tagged release being patched (i.e. `1.6.0`)
-1. Use `git cherry-pick <commit>` to incorporate specific changes required for the minor release
+### Patch
+
+Occasionally it might be pertinent to create a patch release that does not contain all of the changes made to the `main` branch since the most recent major or minor release. In order to avoid creating a release that includes changes unrelated to those that are desired, follow the steps below:
```

Comment:
The change to `might` makes the word `Occasionally` redundant.
```suggestion
It might be pertinent to create a patch release that does not contain all of the changes made to the `main` branch since the most recent major or minor release. In order to avoid creating a release that includes changes unrelated to those that are desired, follow the steps below:
```